### PR TITLE
#701 Support for Windows Terminal

### PR DESCRIPTION
**Pull request by CREITZ25**
#701 Support for Windows Terminal

### CODE REVIEW

**Review by hohwille — CHANGES_REQUESTED** [review]
@CREITZ25 thanks for this great PR. I really love your solution as it works smoothly.
I added one last review comment to rework.

File: scripts/src/main/resources/setup.bat [hunk 1]
```diff
@@ -7,6 +7,11 @@ call scripts\devon.bat ide setup %*
 reg import system/windows/cmd/devon-cmd.reg
 reg import system/windows/power-shell/devon-power-shell.reg
 
+dir %USERPROFILE%\AppData\Local\Microsoft\WindowsApps\wt1.exe >NUL 2>NUL
```
[the remark this answers is not in the document]
Comment (CREITZ25) [reply scripts/src/main/resources/setup.bat]: Oh, I missed this file, sorry

[the remark this answers is not in the document]
Comment (CREITZ25) [reply scripts/src/main/resources/setup.bat]: I corrected it now

[the remark this answers is not in the document]
Comment (hohwille) [reply scripts/src/main/resources/setup.bat]: Great. Ready for merge now.

**Pull request merged**